pkg/logger/logger.go

```diff
@@ -0,0 +1,58 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type LogOptions struct {
+	// sealer log file path, default log directory is `/var/lib/sealer/log`
+	OutputPath string
+	// Verbose: sealer log level,if it is ture will set debug log mode.
+	Verbose bool
+	// DisableColor if true will disable outputting colors.
+	DisableColor         bool
```

Comment:
I think only the terminal output can display content color. So color is a config of console output, rather than a config of LogConfig. So I suggest that we could remove `DisableColor` from the LogOptions.